util/kube/ctl.go

```diff
@@ -247,33 +246,12 @@ func runKubectl(kubeconfigPath string, namespace string, args []string, manifest
 		cmdArgs = append(cmdArgs, "--dry-run")
 	}
 	cmd := exec.Command("kubectl", cmdArgs...)
-	log.Info(cmd.Args)
-	if log.IsLevelEnabled(log.DebugLevel) {
```

Comment:
no need for this

Reply:
We still need this. This is printing the object that we are about to apply. That's not part of CLI command or outputs.